frontend/src/scenes/onboarding/Personalization.tsx

```diff
@@ -7,36 +7,31 @@ import { RadioSelect } from 'lib/components/RadioSelect'
 import { ROLES, PRODUCTS, IS_TECHNICAL } from './personalizationOptions'
 import { Link } from 'lib/components/Link'
 import './Personalization.scss'
-import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 
 export const Personalization = hot(_Personalization)
 
 function _Personalization(): JSX.Element {
-    const { step } = useValues(personalizationLogic)
-
-    // TODO: Redirect if personalization has been completed
-
     return (
         <Row className="personalization-screen">
-            <Col xs={24}>{step === null && <StepOne />}</Col>
+            <Col xs={24}>
+                <StepOne />
```

Comment:
At this point if you're removing all the logic to support multiple steps, which I think is fine for now, then maybe just get rid of the `StepOne` component too?